packages/volto/src/components/manage/Toolbar/Toolbar.jsx

```diff
@@ -220,6 +231,9 @@ class Toolbar extends Component {
     };
     this.props.setExpandedToolbar(this.state.expanded);
     document.addEventListener('mousedown', this.handleClickOutside, false);
+
+    const { userId, getUser } = this.props;
+    getUser(userId);
```

Comment:
@Wagner3UB Did you test to see what would happen if you logged in as one user who does have permission, logged out, then logged in as a different user who doesn't have permission?
I would assume the state gets updated elsewhere and handled by the rendering of this component, but it would be a good thing to check.

Reply:
Yes, no problems with that.